app/src/main/java/com/iron/espresso/presentation/home/study/adapter/viewholder/StudyCategoryViewHolder.kt

```diff
@@ -18,19 +17,16 @@ class StudyCategoryViewHolder(parent: ViewGroup) :
     private val binding =
         DataBindingUtil.bind<com.iron.espresso.databinding.ItemStudyCategoryBinding>(itemView)
 
-    fun bind(item: StudyCategoryItem, listener: StudyCategoryAdapterListener) {
-        binding?.run {
-            setVariable(BR.studyCategoryItem, item)
-            executePendingBindings()
-        }
+    fun bind(item: String, listener: StudyCategoryAdapterListener) {
```

Comment:
```suggestion
    fun bind(item: String) {
```
listener: StudyCategoryAdapterListener 얘는 생성자에 넣는게 좋아보임

Reply:
bind 할 때 마다 listener를 부르는 것이 아닌 생성할 때 넣어주었습니다
1ae3d6d